### ISSUE TITLE
Ensure python mysql module is installed

### ISSUE THREAD

**Issue by afred** [WGBH/mla-playbooks#36]
Was getting this failure:

```
TASK: [geerlingguy.mysql | Remove anonymous MySQL users.] ********************* 
failed: [52.90.157.31] => (item=ip-172-30-0-180) => {"failed": true, "item": "ip-172-30-0-180"}
msg: the python mysqldb module is required
```

Found this issue: https://github.com/geerlingguy/ansible-role-mysql/issues/42

Tested it out by manually adding the MySQL-python module, and rerunning the playbook. It worked.

**Done when** openvault playbook installs `MySQL-python` module before trying to run the task `geerlingguy.mysql | Remove anonymous MySQL users.`

**Comment by afred** [comment]
closed by #37

**Comment by afred** [comment]
This is not fixed. I had gotten a false positive before, by not running it on a new instance.

You cannot simply install `MySQL-python` via `pip` before running the `geerlingguy.mysql` role. It will fail because of a dependence on `mysql_config` (which will be installed in `geerlingguy.mysql` role, and which is why after running it twice on the same instance, it worked, resulting in the false positive).

I think [this line](https://github.com/geerlingguy/ansible-role-mysql/blob/1.9.0/tasks/setup-RedHat.yml#L8) needs to be changed to use `pip` instead of `yum`. Trying.. stand by.

**Comment by afred** [comment]
turns out, we just won't be using mysql. Closed by #46
